ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue

```diff
@@ -40,91 +40,138 @@
 
         <tr v-else-if="row.item.id" :key="row.item.id" class="owner-info" :data-test-id="`owner-info-${row.item.id}`">
           <td class="owner-name">
-            <div v-if="row.item.individualName" class="owner-icon-name">
-              <v-icon class="mr-2">mdi-account</v-icon>
-              <div class="font-weight-bold">
-                {{ row.item.individualName.first }}
-                {{ row.item.individualName.middle }}
-                {{ row.item.individualName.last }}
+            <div :class="{'removed-owner': isRemovedHomeOwner(row.item)}">
```

Comment:
Can we apply `removed-owner` class to the whole row instead? Is it because of `Delete` badge that needs to be in full colour?

Reply:
We could but then have to override on the Badges and Actions explicitly as well. 
So might as well just apply the styling explicitly to the text.